notebooks/demo.ipynb

```diff
@@ -29,7 +29,7 @@
     "import numpy as np\n",
     "from glidertest import fetchers\n",
     "from glidertest import tools\n",
-    "import warnings\n",
+    "#import warnings\n",
```

Comment:
we can probably get rid of this warning mute now that you've fixed it in the tools themselves?